<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-TTL6LUL" name="AutomationTest.PhoneTest" tests="20" failures="1" timestamp="2021-03-10T10:09:17 ICT" time="38.681" errors="0">
  <testcase name="Add_71" time="1.853" classname="AutomationTest.PhoneTest"/>
  <system-out/>
  <testcase name="Add_72" time="1.909" classname="AutomationTest.PhoneTest"/>
  <system-out/>
  <testcase name="Add_73" time="1.907" classname="AutomationTest.PhoneTest"/>
  <system-out/>
  <testcase name="Add_74" time="1.966" classname="AutomationTest.PhoneTest"/>
  <system-out/>
  <testcase name="Add_75" time="1.991" classname="AutomationTest.PhoneTest"/>
  <system-out/>
  <testcase name="Add_76" time="1.930" classname="AutomationTest.PhoneTest"/>
  <system-out/>
  <testcase name="Add_77" time="1.855" classname="AutomationTest.PhoneTest"/>
  <system-out/>
  <testcase name="Add_78" time="1.929" classname="AutomationTest.PhoneTest"/>
  <system-out/>
  <testcase name="Add_79" time="1.866" classname="AutomationTest.PhoneTest"/>
  <system-out/>
  <testcase name="Add_80" time="1.979" classname="AutomationTest.PhoneTest"/>
  <system-out/>
  <testcase name="Add_81" time="1.884" classname="AutomationTest.PhoneTest"/>
  <system-out/>
  <testcase name="Add_82" time="1.932" classname="AutomationTest.PhoneTest"/>
  <system-out/>
  <testcase name="Add_83" time="1.896" classname="AutomationTest.PhoneTest"/>
  <system-out/>
  <testcase name="Add_84" time="1.896" classname="AutomationTest.PhoneTest"/>
  <system-out/>
  <testcase name="Add_85" time="1.946" classname="AutomationTest.PhoneTest"/>
  <system-out/>
  <testcase name="Add_86" time="2.009" classname="AutomationTest.PhoneTest"/>
  <system-out/>
  <testcase name="Add_87" time="2.011" classname="AutomationTest.PhoneTest"/>
  <system-out/>
  <testcase name="Add_88" time="1.937" classname="AutomationTest.PhoneTest"/>
  <system-out/>
  <testcase name="Add_89" time="1.918" classname="AutomationTest.PhoneTest"/>
  <system-out/>
  <testcase name="Add_90" time="2.067" classname="AutomationTest.PhoneTest">
    <failure type="java.lang.AssertionError" message="expected [Please match the requested format.] but found []">
      <![CDATA[java.lang.AssertionError: expected [Please match the requested format.] but found []
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at AutomationTest.PhoneTest.Add_90(PhoneTest.java:340)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1213)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Add_90 -->
  <system-out/>
</testsuite> <!-- AutomationTest.PhoneTest -->
